zipline/algorithm.py

```diff
@@ -1441,6 +1438,13 @@ def _calculate_order(self, asset, amount,
                                                         style)
         return amount, style
 
+    @staticmethod
+    def _round_order(amount):
```

Comment:
Since the intent of this change is to make this overridable by subclasses, can we make this a non-underscore method and put a docstring on it?